### DISCUSSION TITLE
Authorization and authentication to the other micro services

### DISCUSSION THREAD

**Discussion by kumararvindibs** [keycloak/keycloak#21620] [Q&A]
I have user management services with keycloak and authentication token as well (JWT token).
Now I want to authorize and authenticate the other microservices with this token.

Can anyone suggest how to authorize the other microservices?

Looking for your support.

**Comment by christian-2** [comment]
What did you try so far?

**Comment by kumararvindibs** [comment]
Thanks for your response.

I didn't find anything as of now. Neither any documents nor any code
example. I have read somewhere that Keycloak is working as a IAM (Identity
and access management). Please let me know if there is anything so that I
can move forward.

Regards,
Arvind Kumar

**Comment by christian-2** [comment]
Well, the other microservices will have to check the JWT for claims they require for authorization, check times, and also check whether they trust the JWT's signer. You'll usually get library support for all that depending on your platform. It's really a standard pattern and not specific to Keycloak.
